src/pynwb/tests/integration/test_photometry.py

```diff
@@ -136,7 +136,6 @@ def test_roundtrip(self):
             photodetector=0,
             fluorophores=[0],
             location='my location',
-            notes='notes'
```

Comment:
Strictly speaking there should be 

1) a test that you _can_ write the `notes` field successfully and it ends up in the neurodata type or output NWB file

2) a test that you can skip the field and it doesn't become populated in the resulting neurodata type in memory or written to the output NWB file

Right now there only seems to be the latter

Reply:
None of the other optional fields (fiber model number, dichroic_model_number, etc.) are tested in this way.  And all the tests in the unit folder are commented out for some reason.  So short of a full overhaul of the testing suite (which seems way beyond the scope of this fix), I think just omitting 'notes' is the way to do it.

Reply:
> None of the other optional fields (fiber model number, dichroic_model_number, etc.) are tested in the way you described. And all the tests in the unit folder are commented out for some reason. So short of a full overhaul of the testing suite (which seems way beyond the scope of this fix), I think just omitting 'notes' is the way to do it.

None of those underwent a change after initial creation, and this is also our first time giving this attention

A simple unit test should be quick and easy to put together, and never hurts anything